std_msgs/msg/Empty.msg

```diff
@@ -0,0 +1,3 @@
+# Temporary dummy field to get around the fact that some IDL parsers (e.g.,
+# OpenSplice) don't like having empty messages
+uint8 dummy
```

Comment:
Indeed, it's not needed! I had a compile error for Empty.msg that I attributed to its emptiness, but I must have been had some other problem at the time. I just tried with it being empty and that worked fine.